packages/terra-application-utility/src/utility/_UtilityMenuItem.scss

```diff
@@ -0,0 +1,142 @@
+:local {
+  .header-utility-body-item {
+    background-color: var(--terra-application-header-utility-menu-body-item-background-color);
+    color: var(--terra-application-header-utility-menu-body-item-font-color);
+    cursor: pointer;
+    font-family: inherit;
+    font-size: var(--terra-application-header-utility-menu-body-item-font-size, 1rem);
+    list-style-type: none;
+    margin-bottom: var(--terra-application-header-utility-menu-body-item-margin-bottom, 0.3571rem);
+    margin-top: var(--terra-application-header-utility-menu-body-item-margin-top, 0.3571rem);
+    max-width: 100%;
+    overflow-wrap: break-word;
+    padding-bottom: var(--terra-application-header-utility-menu-body-item-padding-bottom, 0.3571rem);
+    padding-left: var(--terra-application-header-utility-menu-body-item-padding-left, 0.2143rem);
+    padding-top: var(--terra-application-header-utility-menu-body-item-padding-top, 0.3571rem);
+    text-overflow: ellipsis;
+    white-space: normal;
+    word-break: break-word;
+    word-wrap: break-word;
+
+    .default-left-inset {
+      margin-left: var(--terra-application-header-utility-menu-body-item-left-inset, 0.6429rem);
+    }
+
+    .default-right-inset {
+      margin-right: var(--terra-application-header-utility-menu-body-item-right-inset, 0.6429rem);
+    }
+
+    &:hover {
+      background-color: var(--terra-application-header-utility-menu-body-item-hover-background-color, #ebf6fd);
+    }
+
+    &:focus {
+      box-shadow: var(--terra-application-header-utility-menu-body-item-focus-box-shadow, 0 0 1px 3px rgba(76, 178, 233, 0.5), 0 0 7px 4px rgba(76, 178, 233, 0.35));
+    }
+
+    .checkmark {
+      flex-shrink: 0;
+      margin-left: var(--terra-application-header-utility-menu-body-item-checkmark-margin-left, 0.2857rem);
+      margin-right: var(--terra-application-header-utility-menu-body-item-checkmark-margin-right, 0.5rem);
+      visibility: hidden;
+      width: var(--terra-application-header-utility-menu-body-item-checkmark-width, 0.7143rem);
+    }
+
+    .selected.checkmark {
+      visibility: visible;
+    }
+
+    .chevron {
+      margin-right: var(--terra-application-header-utility-menu-body-item-chevron-margin-right, 0.2143rem);
+      visibility: hidden;
+      width: var(--terra-application-header-utility-menu-body-item-chevron-width, 0.7143rem);
+    }
+
+    .chevron.has-chevron {
```

Comment:
? Why these visibility differences on the items?

Reply:
To toggle the checkmark and keep the left/right insets consistent for all menu items - if a single item is selectable or has a chevron, all the items need to consistently align.

Reply:
![smart](https://media.giphy.com/media/d3mlE7uhX8KFgEmY/giphy.gif)